docs/Tutorials/Contracts/Calling-Contract-Functions.md

```diff
@@ -93,7 +93,7 @@ To verify that a value has been updated, perform a `get` call after a `set` upda
 
 This private contracts example uses the same `SimpleStorage.sol` contract as in the
 [public contracts example](#interact-with-public-contracts), but it uses the
-[web3js-eea](https://github.com/ConsenSys/web3js-eea) library and the
+[web3js-quorum](https://consensys.github.io/web3js-quorum/latest/index.html) library and the
```

Comment:
non-blocking: just wanted to point out that the previous link was pointing to the github repo and the new one points to the documentations